test/jdk/javax/swing/JColorChooser/Test6977726.java

```diff
@@ -41,7 +41,9 @@ public static void main(String[] args) throws Exception {
         String instructions = """
                 Check that there is a panel with "Text Preview Panel" text
                 and with title "Preview" in the JColorChooser.
-                Test passes if the panel is as described, test fails otherwise.""";
+                Test passes if the panel is as described, test fails otherwise.
+
+                Note: "Preview" title is not applicable for GTK Look and Feel.""";
```

Comment:
The test summary says it tests `setPreviewPanel`...if it's not present in native, then the full test is not applicable for GTK..I guess you can check for UIManager.getLookAndFeel() is GTK, then bail out..

Reply:
> The test summary says it tests `setPreviewPanel`...if it's not present in native, then the full test is not applicable for GTK..

Actually the PreviewPanel is added for all L&F in JDK implementation in BasicColorChooserUI ( see here [L147](https://github.com/kumarabhi006/jdk/blob/70f3e99016311a6520e6a7c0da4c7ff718364976/src/java.desktop/share/classes/javax/swing/plaf/basic/BasicColorChooserUI.java#L147) and [L365](https://github.com/kumarabhi006/jdk/blob/70f3e99016311a6520e6a7c0da4c7ff718364976/src/java.desktop/share/classes/javax/swing/plaf/basic/BasicColorChooserUI.java#L365) ).

So, `setPreviewPanel` should be ok for GTK as well.

Reply:
ok...in that case please add a comment in the test citing
"ColorChooser.showPreviewPanelText", Boolean.FALSE for GTKLookAndFeel because of which it is not applicable, in case we need to revisit this in future, 
although it does not make sense that preview panel is supported but not preview panel text, but let maintain status-quo and not change product property just because of recently-modified internal test

Reply:
> "ColorChooser.showPreviewPanelText", Boolean.FALSE for GTKLookAndFeel because of which it is not applicable, in case we need to revisit this in future,

I guess it is better to add the comment in JBS in stead of the test. As a part of test history it can be referenced in future.
What do you say?

> although it does not make sense that preview panel is supported but not preview panel text, 

Yeah that was the trigger point for me to change the property as it doesn't make any sense to have a preview panel but not the title. 

> but let's maintain status-quo and not change product property just because of recently-modified internal test

seems fine.